### PR TITLE
Midterm blog by SanchiMittal

### PR DESCRIPTION
**Pull request by SanchiMittal**
Midterm blog added.

cc @lmoneta

### CODE REVIEW

**Review by sanjibansg — APPROVED** [review]

File: _gsocblogs/2022/blog_ROOT_SanchiMittal.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+project: ROOT - Machine Learning Developments
+title: Batch Generator for training machine learning models
+author: Sanchi Mittal
+photo: images/blog_authors/SanchiMittal.jpeg
```
Comment (agheata) [inline _gsocblogs/2022/blog_ROOT_SanchiMittal.md:5]:
```suggestion
photo: blog_authors/SanchiMittal.jpeg 
```

File: _gsocblogs/2022/blog_ROOT_SanchiMittal.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+intro: |
+   Develop a generator in C++ and Python to read data from the ROOT I/O and input them to the Python machine learning tools such as Tensorflow/Keras and PyTorch. 
+---
+
+
+# Sanchi's Midterm Blog for GSoC 2022
+
+Through this blog, I'll be going over the work that I have done by this middle point in time as a contributor for GSoC 2022 program with CERN-HSF under the project "ROOT - Machine Learning Developments : Batch Generator for training machine learning models".
+
+![CERN-HSF-GSoC-logo](https://hepsoftwarefoundation.org/images/CERN-HSF-GSoC-logo.png)
```
Comment (agheata) [inline _gsocblogs/2022/blog_ROOT_SanchiMittal.md:19]:
```suggestion
![CERN]({{ site.baseurl }}/images/CERN-HSF-GSoC-logo.png){:height="100px"} Google Summer of Code 2022
```

**Review by agheata — CHANGES_REQUESTED** [review]
@SanchiMittal please adopt the changes I suggested

**Comment by SanchiMittal** [comment]
Thank you for the review! I have updated the PR with requested changes.

**Review by agheata — APPROVED** [review]

**Pull request merged**